tests/test_quantum_gradients.py

```diff
@@ -142,16 +142,22 @@ def test_cv_gradients_gaussian_circuit(self):
         """Tests that the gradients of circuits of gaussian gates match between the finite difference and analytic methods."""
         self.logTestName()
 
-        class PolyN(qml.expval.PolyXP):
+        class PolyN(qml.ops.PolyXP):
             "Mimics MeanPhoton using the arbitrary 2nd order observable interface. Results should be identical."
             def __init__(self, wires):
                 hbar = 2
                 q = np.diag([-0.5, 0.5/hbar, 0.5/hbar])
                 super().__init__(q, wires=wires)
                 self.name = 'PolyXP'
 
-        gates = [cls for cls in qml.ops.cv.all_ops if cls.supports_analytic]
-        obs   = [qml.expval.X, qml.expval.MeanPhoton, PolyN]
+        gates = []
+        for name in qml.ops._cv__ops__:
```

Comment:
how come this was necessary?

Reply:
I defined `_cv__ops__` to be a set of gate names as strings, instead of classes, based on the other tests I fixed up. Worked well until I got to this test, and realized it was the only one that relied on it actually being a list of classes 😭